docs: Project api token (#3405)
Unleash/unleash: website/docs/reference/api-tokens-and-client-keys.mdx

Use API tokens to connect to the Unleash server API. API tokens come in four distinct types:

-   [Admin tokens](#admin-tokens)

> -   [Project level access tokens](#project-access-tokens)

Reviewer: It's not a different type of API token, so I don't think it should be listed here.

```suggestion
```

Support for scoped admin tokens with more fine-grained permissions is currently in the planning stage.

> ### Project level access tokens
> 
> All project members can see any `CLIENT` (server-side client token) and `FRONTEND` (client-side client token) respectively.

Reviewer: Thanks! Looking at this, I think we could reword it to instead be about visibility. But I need you to help me out with the details here: how visible are tokens usually? I've got a suggestion below, but it needs to be modified based on what is the reality. I'd be happy to help rewrite this again, but I don't quite know how all the permissions around API tokens work these days.

Oh, and I also think this should be moved to before the admin tokens section.

```suggestion
### API token visibility

:::info project-level visibility

Project-level visibility and access to API tokens was introduced in Unleash 4.22.

:::

By default, only admin users can create API tokens, and only admins can see their values.

However, any [client](#client-tokens client tokens) and [front-end tokens](#front-end-tokens) that are applicable to a project, will also be visible to any members of that project that have the `READ_PROJECT_API_TOKEN` permission (all project members by default). 

Similarly, any project members with the `CREATE_PROJECT_API_TOKEN` permission can also create client and front-end tokens for that specific project ([how to create project API tokens](../how-to/how-to-create-project-api-tokens.mdx)).
```

Author: agreed and well done

Use personal access tokens to:

> - Provide more fine-grained permissions for automation than an admin token provides

Reviewer: Looks like you forgot to remove this line when copying the PAT section 💁🏼 

```suggestion
```

- Give access to all project members (or anyone with `READ_PROJECT_API_TOKEN` permission) to read the token

> - Give access to the project owner (or anyone with `CREATE_PROJECT_API_TOKEN` permission) to create a token,

Reviewer: If you agree with my suggestion regarding visibility, then I think we can also delete this and the rest of the other content from this section. 

But before you do: isn't the `read_project_api_token` permission a project-level permission? So don't you have to be BOTH a project member AND have that permission? And the same for creating.

Author: Yes, we assign the `read_project_api_token` permission to Project Members by default and the same for project owner and `create_project_api_token`
